test/extensions/filters/http/common/fuzz/uber_per_filter.cc

```diff
@@ -70,11 +73,39 @@ void UberFilterFuzzer::guideAnyProtoType(test::fuzz::HttpData* mutable_data, uin
   mutable_any->set_type_url(type_url);
 }
 
+void cleanAttachmentTemplate(Protobuf::Message* message) {
+  envoy::extensions::filters::http::squash::v3::Squash& config =
+      dynamic_cast<envoy::extensions::filters::http::squash::v3::Squash&>(*message);
+  std::string json;
+  Protobuf::util::JsonPrintOptions json_options;
+  if (!Protobuf::util::MessageToJsonString(config.attachment_template(), &json, json_options)
```

Comment:
@yuval-k do you think the Squash filter should do some validation here? Today the control plane is trusted, but we'd like to at least have the property that a bad config can't crash Envoy.